src/js/rx/config.js

```diff
@@ -71,20 +73,5 @@ module.exports = {
     suspended: 'Active - Suspended',
     refillinprocess: 'In process',
     unknown: 'Unknown'
-  },
-
-  sortOptions: [
-    {
-      value: 'prescriptionName',
-      label: 'Prescription name'
-    },
-    {
-      value: 'facilityName',
-      label: 'Facility name'
-    },
-    {
-      value: 'lastSubmitDate',
-      label: 'Last submit date'
-    }
-  ]
```

Comment:
Specific to the cards in active view. Moved it to `PrescriptionList`, the only place where it would be used.